src/components/structures/ThreadView.tsx

```diff
@@ -188,13 +189,32 @@ export default class ThreadView extends React.Component<IProps, IState> {
         }
     };
 
+    private updateThreadRelation = (): void => {
+        this.setState({
+            lastReply: this.state.thread?.lastReply((ev: MatrixEvent) => {
+                return ev.isRelation(THREAD_RELATION_TYPE.name) && !ev.status;
+            }),
+        });
+    };
+
     private updateThread = (thread?: Thread) => {
+        if (this.state.thread) {
+            this.state.thread.off(ThreadEvent.NewReply, this.updateThreadRelation);
+            this.props.room.off(RoomEvent.LocalEchoUpdated, this.updateThreadRelation);
+        }
+
         if (thread && this.state.thread !== thread) {
+            thread.on(ThreadEvent.NewReply, this.updateThreadRelation);
+            this.props.room.on(RoomEvent.LocalEchoUpdated, this.updateThreadRelation);
```

Comment:
If `updateThread()` gets called with `this.state.thread`, we'll disable the updates and not re-enable them; is that the desired behavior?

Reply:
That's not the intended behaviour indeed. I've updated this scenario, cheers!